### ISSUE TITLE
Issue with png and jpg files when there are the same name.

### ISSUE THREAD

**Issue by kadetth** [yireo/Yireo_Webp2#169]
If in cache is two images for two produtcts with the same name like but another extension like test.jpg and test.png the extension make one webp image like test.webp. In this case one product display wrong image.

**Comment by Bashev** [comment]
Do you try to change "Add Image Hash" -> "Yes"

![image](https://github.com/user-attachments/assets/66d8b95b-22ed-460e-b58e-1652ba68fd58)

**Comment by kadetth** [comment]
No, I missed this option :) That should be solve the problem. Thank You.

**Comment by jissereitsma** [comment]
I assume this issue has been resolved by now. Feel free to re-open this ticket if it has not.
